### ISSUE TITLE
typeparam indistinguishable from param

### ISSUE THREAD

**Issue by KyleKolander** [MrJul/ReSharper.EnhancedTooltip#189]
When I have a function with XML Doc comments that include both typeparam and param, each of them show up in the enhanced tooltip as Parameter, so it's difficult (or at least not obvious) whether one of those Parameter lines is for an actual function parameter or for a generic type parameter.  If this is already implemented, could you please reply with the setting(s) to change so this works as described here.  Otherwise, could you implement it so that typeparam is displayed as Type or Generic Type or something like that, rather than Parameter?

**Comment by DimitarCC** [comment]
Can you please show a screenshot of that? I am not sure what exactly is the problem...

**Comment by KyleKolander** [comment]
![ReSharper EnhancedTooltip_001](https://github.com/MrJul/ReSharper.EnhancedTooltip/assets/22461822/10fb7de5-bab6-4a87-ab40-34b98f4e377a)

![ReSharper EnhancedTooltip_002](https://github.com/MrJul/ReSharper.EnhancedTooltip/assets/22461822/9c09d813-9f5b-4a35-991d-16a5790fac05)

There you can see that the first two parameters (in the XML Doc comment) are `typeparam`, whereas all the others are just `param`.  The enhanced tooltip shows all of them as if they are `param`, when in reality, those first two `typeparam` are not function parameters, they are generic types.

**Comment by DimitarCC** [comment]
OK the problem is that long labels are not very suitable to display in the tooltip...
So if i put Type Parameter as a label it will be quite long..... But i am not sure what else it can be so i may try it to see how it will look like...

**Comment by KyleKolander** [comment]
Any luck with tis @DimitarCC ?  How about instead of `Parameter`, just put `Type`?  That would clearly distinguish it from the method parameters while also keeping it as a short label...

**Comment by DimitarCC** [comment]
Yes there is already a version displaying type params differently...

![image](https://github.com/MrJul/ReSharper.EnhancedTooltip/assets/1904901/fa029757-040d-4678-afc6-9614813edf53)

Use the attached nuget package and install it locally with enabled prerelease extensions.
[JLebosquain.EnhancedTooltip.3.25.1-pre.nupkg.zip](https://github.com/MrJul/ReSharper.EnhancedTooltip/files/15494983/JLebosquain.EnhancedTooltip.3.25.1-pre.nupkg.zip)

**Comment by KyleKolander** [comment]
Cool! Either way is a helpful enhancement!
